common/types/shared.go

```diff
@@ -4636,13 +4636,22 @@ func (v *GetTaskListsByDomainRequest) GetDomain() (o string) {
 
 // GetTaskListsByDomainResponse is an internal type (TBD...)
 type GetTaskListsByDomainResponse struct {
-	TaskListNames []string `json:"taskListNames,omitempty"`
```

Comment:
Just want to confirm that making breaking changes here is ok as no users (internal/open source) are actively using it?

Reply:
Yes. No user is using this